contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -574,6 +579,41 @@ contract BosonVoucherBase is IBosonVoucher, BeaconClientBase, OwnableUpgradeable
         _setContractURI(_newContractURI);
     }
 
+    /** @notice Make a call to an external contract.
+     *
+     * Reverts if:
+     * - _to is zero address
+     * - call to external contract fails
+     * - caller is not the owner
+     *
+     * @param _to - address of the contract to call
+     * @param _data - data to pass to the external contract
+     */
+    function callExternalContract(address _to, bytes memory _data) external payable onlyOwner {
+        require(_to != address(0), INVALID_ADDRESS);
+
+        _to.functionCallWithValue(_data, msg.value, FUNCTION_CALL_NOT_SUCCESSFUL);
```

Comment:
this is weirding me out, but this might be some syntax that i am not aware of ... 

When i look at Address.sol from OZ https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.8.1/contracts/utils/Address.sol#L85-L101 

The method we are using seems to have more parameters, what is going on here ?

Reply:
I am guessing that the _to.functionCallWith ... the to gets wrangled into the first parameter "target" ... It just does my head in a bit, and doesn't seem that readable :)

Reply:
It is great that we can use OZ here...